tyssue/draw/plt_draw.py

```diff
@@ -167,11 +167,125 @@ def sheet_view(sheet, coords=COORDS, ax=None, **draw_specs_kw):
+
+    {"edge": {
+	"visible": true,
+    "width": 0.5,
+    "head_width": 0.0,
+    "length_includes_head": true,
+    "shape": "right",
+    "color": "#2b5d0a",
+    "alpha": 0.8,
+    "zorder": 1,
+	"colormap": "viridis"
+    },
+    "vert": {
+    "visible": false,
+    "s": 100,
+    "color": "#000a4b",
+    "alpha": 0.3,
+    "zorder": 2
+    },
+    "grad": {
+    "color":"#000a4b",
+    "alpha":0.5,
+    "width":0.04
+    },
+    "face": {
+    "visible": false,
+    "color":"#8aa678",
+    "alpha": 1.0,
+    "zorder": -1
+    },
+    "axis": {
+    "autoscale": true,
+    "color_bar_cmap":"viridis",
+    "color_bar_range":false, 
+    "color_bar_label":false, 
+    "color_bar_target":"face"}
+    }
+    """
+    draw_specs = sheet_spec()
+    spec_updater(draw_specs, draw_specs_kw)
+    #    if (ax1 is None) or (ax2 is None):
+    #        fig, (ax1,ax2) = plt.subplots(1,2,gridspec_kw={'width_ratios': [100, 1]})
+    #    else:
+    #        fig, (ax1,ax2) = plt.subplots(1,2,gridspec_kw={'width_ratios': [100, 1]})
+    if (ax1 is None) or (ax2 is None):
+        fig = plt.figure()
+    else:
+        fig = plt.get_figure()
+
+    grid0 = plt.GridSpec(10, 10)
+    grid0.update(wspace=0.0)
+
+    ax1 = fig.add_subplot(grid0[:, :9])
+
+    vert_spec = draw_specs["vert"]
+    if vert_spec["visible"]:
+        ax1 = draw_vert(sheet, coords, ax1, **vert_spec)
+
+    edge_spec = draw_specs["edge"]
+    if edge_spec["visible"]:
+        ax1 = draw_edge(sheet, coords, ax1, **edge_spec)
+
+    face_spec = draw_specs["face"]
+    if face_spec["visible"]:
+        ax1 = draw_face(sheet, coords, ax1, **face_spec)
+
+    axis_spec = draw_specs["axis"]
+    if axis_spec["autoscale"] == True:
+        ax1.autoscale()
+        ax1.set_aspect("equal")
+    else:
+        ax1.set_xlim(axis_spec["x_min"], axis_spec["x_max"])
+        ax1.set_ylim(axis_spec["y_min"], axis_spec["y_max"])
+        ax1.set_aspect("equal")
+
+    # gc
+    ax2 = fig.add_subplot(grid0[:, 9])
+    cmap = cm.get_cmap(draw_specs["axis"]["color_bar_cmap"])
+
+    if draw_specs["axis"]["color_bar_range"] == False:
+        if draw_specs["axis"]["color_bar_target"] == "face":
+            norm = mpl.colors.Normalize(
+                vmin=np.min(sheet.face_df["col"]), vmax=np.max(sheet.face_df["col"])
+            )
+        else:
+            print(
+                "automatic color bar range only works for the face dataframe use draw_specs dictionary under axis, color_bar_range entry to specify a range"
+            )
+    else:
+        norm = mpl.colors.Normalize(
+            vmin=draw_specs["axis"]["color_bar_range"][0],
+            vmax=draw_specs["axis"]["color_bar_range"][1],
+        )
+
+    cb1 = mpl.colorbar.ColorbarBase(ax2, cmap=cmap, norm=norm, orientation="vertical")
+    if draw_specs["axis"]["color_bar_label"] == False:
+        cb1.set_label("a.u.")
+    else:
+        cb1.set_label(draw_specs["axis"]["color_bar_label"])
+    return fig, ax1, ax2
+
```

Comment:
This is cool :smile: 

Do you think you can add the colorbar functionality directly to the  `sheet_view` function? Using the `get` method as I suggested should make it backward compatible, and it would help to adopt it in other code bases.